test/views/SummarySubscriptionsViewSpec.scala

```diff
@@ -39,18 +40,37 @@ class SummarySubscriptionsViewSpec extends ViewBehaviours with SummarySubscripti
       getTaxYear(CurrentYear) -> 300,
       getTaxYear(CurrentYearMinus1) -> 0)
 
-    val applyView = view.apply(
-      subscriptions = subscriptions,
-      npsData = npsData,
-      nextPageUrl = navigator.nextPage(SummarySubscriptionsPage, NormalMode, userAnswersCurrentAndPrevious).url,
-      mode = NormalMode
-    )(fakeRequest, messages)
+    def applyView(arePsubsEmpty: Boolean = true): Html = {
+      view.apply(
+        subscriptions = subscriptions,
+        npsData = npsData,
+        nextPageUrl = navigator.nextPage(SummarySubscriptionsPage, NormalMode, userAnswersCurrentAndPrevious).url,
```

Comment:
Moved the tests into the component as it already tests both sides (true and false)

Reply:
Agreed, I have changed the tests to be similar to what is seen in controllers with the focus being on the navigator rather than the view for testing the correct nav route